contracts/BlendSwap.sol

```diff
@@ -8,7 +8,7 @@ contract BlendSwap {
     mapping (bytes32 => mapping (address => Swap)) public swaps;
 
     IERC20 public blend;
-    uint public constant TEN_MINUTES = 600000;
+    uint public constant TEN_MINUTES = 600;
```

Comment:
FYI, you can just write `10 minutes` https://docs.soliditylang.org/en/v0.5.3/units-and-global-variables.html#time-units

Reply:
@sashasashasasha151 let's follow Kirill's proposal